ThunderCloud/AchievementDisplayView.swift

```diff
@@ -56,6 +56,8 @@ open class AchievementDisplayView: UIView, AchievementDisplayable {
 		subtitleLabel.textAlignment = .center
 		subtitleLabel.backgroundColor = .clear
 		subtitleLabel.font = titleLabel.font
+        subtitleLabel.isUserInteractionEnabled = false
```

Comment:
Indentation seems a bit strange here. Is that GitHub, or the file itself? :)

Reply:
Possibly the file itself, the whole of ThunderCloud needs a re-indent running at some point since we moved from tabs to spaces!

Reply:
@ryanbourneuk there we go! :P

Reply:
Thank you :D